dockers/docker-base-buster/apt-retries-count

```diff
@@ -0,0 +1 @@
+Acquire::Retries "20";
```

Comment:
what is 20? 20 retries? can we make it configurable? for example, use j2 template and set a variable int he makefile.

Reply:
It is a number of times apt will retry to acquire the package in case of failure. The default is 3. It won't try to acquire more times if the first one will be successful.
Description from [man page](https://manpages.ubuntu.com/manpages/xenial/en/man5/apt.conf.5.html):
```
Retries
    Number of retries to perform. If this is non-zero APT will retry failed files the
    given number of times.
```